nservicebus/nhibernate/outbox_customizing_nhibernate_[7,).partial.md

```diff
@@ -0,0 +1,15 @@
+## Customizing outbox record peristence
+
+By default the outbox records are persisted in the following way:
```

Comment:
The items below are not consistent in terms of formatting. Two are missing the full-stop at the end of the sentence.